core/src/main/java/org/pentaho/platform/web/http/api/resources/services/SchedulerService.java

```diff
@@ -197,84 +198,74 @@ public Job createJob( JobScheduleRequest scheduleRequest )
   /**
    * Wrapper function around {@link SchedulerOutputPathResolver#resolveOutputFilePath()} calls
    * {@link #getSchedulerOutputPathResolver(JobScheduleRequest)} to get instance.
-   * @param scheduleRequest
```

Comment:
We should complete the documentation, or leave it as is for now.

Reply:
I understand that completing the documentation is important, however, in this particular case, I was not fully aware of the details regarding the implementation which could have resulted in inaccurate information. Leaving an empty tag in the documentation should be avoided.